parsers/ZA.py

```diff
@@ -11,9 +11,11 @@
 TIMEZONE = "Africa/Johannesburg"
 
 
-def get_url() -> str:
+def get_url(
+    date_shift: int = 0
+) -> str:
     """Returns the formatted URL"""
-    date = datetime.utcnow()
+    date = datetime.utcnow() - timedelta(weeks = date_shift * 4)
```

Comment:
4 weeks seems like a big step for the retry, would it not make more sense to use something like 1 week or what are the publishing deadline for the underlying data/source?

Reply:
Yes this should work as well, but does not change much, as the link is merged of year an month. The exact date does not matter.

This will also just be needed when in the current month there have not been any data uploaded by the source.

Reply:
Let's leave it as it is then.